server/init.sh

```diff
@@ -12,6 +12,9 @@ git checkout ${BRANCH}
 git pull
 git submodule update
 
+# allow eval script for executing javascript in webview (LoginIT test)
```

Comment:
```suggestion
# allow eval script for executing javascript in webview (LoginIT test)
if test -z "$EVAL"
then
	echo "\$EVAL not set, ignoring..."
else
	sed -i s'/protected $evalScriptAllowed = false;/protected $evalScriptAllowed = true;/' 		 
	lib/public/AppFramework/Http/ContentSecurityPolicy.php
fi
```

Reply:
Where would I have to set "$EVAL"? In drone?

Reply:
in the environment section of the drone job :)

Reply:
Then I'll do this tomorrow.
Very good approach!